src/main/java/hudson/plugins/git/extensions/impl/SubmoduleOption.java

```diff
@@ -45,15 +45,17 @@
     /** Use --depth flag on submodule update command - requires git>=1.8.4 */
     private boolean shallow;
     private Integer depth;
+    private Integer threads = 1;
 
     @DataBoundConstructor
-    public SubmoduleOption(boolean disableSubmodules, boolean recursiveSubmodules, boolean trackingSubmodules, String reference,Integer timeout, boolean parentCredentials) {
+    public SubmoduleOption(boolean disableSubmodules, boolean recursiveSubmodules, boolean trackingSubmodules, String reference, Integer timeout, boolean parentCredentials) {
         this.disableSubmodules = disableSubmodules;
         this.recursiveSubmodules = recursiveSubmodules;
         this.trackingSubmodules = trackingSubmodules;
         this.parentCredentials = parentCredentials;
         this.reference = reference;
         this.timeout = timeout;
+        this.threads = 1;
```

Comment:
There is already a default value set in line 48 so this is not required here. It duplicates code and increases the potential for inconsistencies.

Please remove this line.